.ci_scripts/update_docs

```diff
@@ -2,35 +2,27 @@
 
 set -ex
 
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+BASE_DIR=$(dirname $HERE)
+
 # configure bot account
 if [[ "$CI" == "1" ]]; then
   git config --global user.email "pelson.pub+conda-forge@gmail.com"
   git config --global user.name "conda-forge-admin"
   git checkout -b new_site_content
 fi
 
-python .ci_scripts/generate_cfep_index.py
-
-pushd sphinx/src
+python "$BASE_DIR/.ci_scripts/generate_cfep_index.py"
 
+pushd "$BASE_DIR/sphinx/src"
+make clean
 # -W --keep-going: list all warnings but fail build in case there are any
-# -n: check validity of all links
```

Comment:
I guess we cannot run link check anymore? This has caught many bugs for us.

Reply:
Docusaurus runs it later (and much faster), so no need to do it with Sphinx. Let me push a broken link on purpose to show it in action.

Reply:
No need to test it. I didn't know! Thank you!

Reply:
Well, turns out it only check internal links, no external stuff. Adding a new step for now.